src/main/scala/com/cognite/spark/datasource/CdpRelation.scala

```diff
@@ -46,6 +52,16 @@ abstract class CdpRelation[T: Decoder](config: RelationConfig, shortName: String
 
   def cursors(): Iterator[(Option[String], Option[Int])] = NextCursorIterator(listUrl(), config)
 
+  def deleteItems(apiKeyAuth: Auth, baseUrl: Uri, rows: Seq[Row], maxRetries: Int): IO[Unit] = {
+    val deleteItems: Seq[Long] = rows.map(r => fromRow[DeleteItem](r).id)
+    post(
+      config.auth,
+      uri"${baseUrl}/delete",
```

Comment:
We can remove the curly brackets here since it only references a single value. I see this applies to the function definitions below as well, which is probably why you did it. Great if you can remove the brackets there as well.